### ISSUE TITLE
Strange error about "undefined method `version' for nil:NilClass (NoMethodError)"

### ISSUE THREAD

**Issue by ribosomerocker** [henrycatalinismith/ppl#89]
Hello, I have discovered this project while looking for address books I could use with neomutt - and this seems to be a nice project to go with. But - I seem to have come upon an issue while trying to use this on Artix Linux, based on Arch (which does change the ruby environment by default according to the wiki, which is why I mention this). 

Installation goes smoothly, but after trying to execute the binary:
```
/home/mon/.local/share/gem/ruby/3.0.0/gems/ppl-4.0.3/lib/ppl.rb:6:in `<module:Ppl>': undefined method `version' for nil:NilClass (NoMethodError)
        from /home/mon/.local/share/gem/ruby/3.0.0/gems/ppl-4.0.3/lib/ppl.rb:3:in `<top (required)>'
        from <internal:/usr/lib/ruby/site_ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from <internal:/usr/lib/ruby/site_ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from /home/mon/.local/share/gem/ruby/3.0.0/gems/ppl-4.0.3/bin/ppl:8:in `<top (required)>'
        from /home/mon/.local/share/gem/ruby/3.0.0/bin/ppl:25:in `load'
        from /home/mon/.local/share/gem/ruby/3.0.0/bin/ppl:25:in `<main>'
```

**Comment by rv178** [comment]
Same error here.

**Comment by acant** [comment]
FYI @ribosomerocker and @rv178 if you still need it, I think I have a fix to this bug in #90
